README.md

```diff
@@ -6,9 +6,7 @@
 ![Go](https://github.com/redpanda-data/redpanda/workflows/Go/badge.svg)
 ![C++](https://github.com/redpanda-data/redpanda/workflows/build-test/badge.svg)
 
-[<p align="center"><img src="docs/PANDA_sitting.jpg" alt="redpanda sitting" width="400"/></p>](https://redpanda.com/redpanda)
-<img src="https://static.scarf.sh/a.png?x-pxid=3c187215-e862-4b67-8057-45aa9a779055" />
-<img src="https://www.google-analytics.com/collect?v=2&tid=G-G8V239NH6H&cid=555&t=event&en=page_view&dp=redpanda-data%2Fredpanda&dt=Redpanda%20GitHub" />
+[<p align="center"><img src="docs/PANDA_sitting.png" alt="redpanda sitting" width="400"/></p>](https://redpanda.com/redpanda)
```

Comment:
despite them not working because of caching, did we find out if this were unused and we could remove it?

Reply:
I don't think we ever got confirmation, other than the [Github Docs](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/about-anonymized-urls) indicating as much.

Reply:
@c4milo do you know about these trackers? i think you added the GA one? we were thinking about removing them as it's not clear they work in the readme given that github caches the tracker images.

Reply:
Inspecting element in a web browser shows that the two tracking pixels are most certainly not pointing back to `scarf.sh` or `google-analytics.com`, and instead to `camo.githubusercontent.com`.

FWIW, the scarf tracking pixel is invisible even in dark mode, and it is the google-analytics pixel that is responsible for the visible period in the README.

Reply:
yep. so the question here is not if they work/not, but if they need to be fixed or removed.

Reply:
I think we need to wait for an internal resolution (slack) on this before proceeding.